avm/res/fabric/capacity/main.bicep

```
@@ -0,0 +1,92 @@
+metadata name = 'Fabric Capacities'
+metadata description = 'This module deploys Fabric capacities, which provide the compute resources for all the experiences in Fabric.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Name of the resource to create.')
+param name string
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Tags of the resource.')
+param tags object?
+
+@allowed([
+  'F2'
+  'F4'
+  'F8'
+  'F16'
+  'F32'
+  'F64'
+  'F128'
+  'F256'
+  'F512'
+  'F1024'
+  'F2048'
+])
+@description('Optional. SKU tier of the Fabric resource.')
+param skuName string = 'F2'
+
+@allowed(['Fabric'])
+@description('Optional. SKU name of the Fabric resource.')
+param skuTier string = 'Fabric'
+
+@description('Required. List of admin members. Format: ["something@domain.com"].')
+param adminMembers array
+
+@description('Optional. Enable/Disable usage telemetry for module.')
+param enableTelemetry bool = true
+
+// ============== //
+// Resources      //
+// ============== //
+
+#disable-next-line no-deployments-resources
+resource avmTelemetry 'Microsoft.Resources/deployments@2024-03-01' = if (enableTelemetry) {
+  name: '46d3xbcp.res.fabric-capacity.${replace('-..--..-', '.', '-')}.${substring(uniqueString(deployment().name, location), 0, 4)}'
+  properties: {
+    mode: 'Incremental'
+    template: {
+      '$schema': 'https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#'
+      contentVersion: '1.0.0.0'
+      resources: []
+      outputs: {
+        telemetry: {
+          type: 'String'
+          value: 'For more information, see https://aka.ms/avm/TelemetryInfo'
+        }
+      }
+    }
+  }
+}
+
+resource fabricCapacities 'Microsoft.Fabric/capacities@2023-11-01' = {
```

As per convention I'd suggest a singular form for the symbolic name
```suggestion
resource fabricCapacity 'Microsoft.Fabric/capacities@2023-11-01' = {
```

---

Updated